.github/workflows/ephemeral-instances-pr-comment.yml

```diff
@@ -52,34 +48,14 @@ jobs:
           ref: main
           path: ephemeral
 
-      - name: Setup Go
-        uses: actions/setup-go@v4
+      - name: build and deploy ephemeral instance
+        uses: ./ephemeral
```

Comment:
out of curiosity: this refers to the path in the previous step, right?

Reply:
yes!